devops_sccs/cache.py

```diff
@@ -14,31 +14,35 @@
 
 # You should have received a copy of the GNU Lesser General Public License
 # along with python-devops-sccs.  If not, see <https://www.gnu.org/licenses/>.
-
+from multiprocessing import managers
 import logging
-        
+import multiprocessing
+import threading
+import traceback
+
 class AsyncCache(object):

```

Comment:
```suggestion
class AsyncCache:

```
toutes les classes heritent de `object` par default, pas la peine de l'explicite (je sais que ca a ete fait comme ca un peu partout dans le codebase, mais je pense que c'est redondant) [doc](https://docs.python.org/3/reference/compound_stmts.html#class)